extensions/resteasy-reactive/quarkus-resteasy-reactive/runtime/src/main/java/io/quarkus/resteasy/reactive/server/runtime/MultipartFormHandler.java

```diff
@@ -191,10 +197,15 @@ public void handle(Void v) {
                         MultipartFormVertxHandler.this.deleteFileUploads();
                     }
                 });
+                restoreProperTCCL();
```

Comment:
All these calls look very hackyish to me. And I'm also worried about how things could go with user-developed handlers. Isn't there a more generic way to fix this?

Reply:
No, if there were we would have solved it.

It's essentially the same hack we do in the vertx-web recorder